src/network-manager/azext_network_manager/_params.py

```diff
@@ -536,3 +536,150 @@ def load_arguments(self, _):
         c.argument('flag', type=str, help='Default rule flag.')
         c.argument('kind', type=str, help='Required. Whether the rule is custom or default.Constant filled by server.', arg_type=get_enum_type(['Custom', 'Default']))
         c.ignore('user_rule')
+
+    with self.argument_context('network manager connection create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('network_manager_connection_name', options_list=['--name','-n','--network-manager-connection-name'],
+                   type=str, help='The name of the network manager connection.')
+        c.argument('network_manager_id', type=str, help='The id of the network manager')
```

Comment:
we should support both name or ID as a value. The transformation from name to ID could be implemented in validator. You can follow this example in azure-cli:

https://github.com/Azure/azure-cli/blob/43d0937be564a3a75787c574102106e247f3b078/src/azure-cli/azure/cli/command_modules/network/_params.py#L903

https://github.com/Azure/azure-cli/blob/43d0937be564a3a75787c574102106e247f3b078/src/azure-cli/azure/cli/command_modules/network/_validators.py#L733-L741